README.md

```diff
@@ -1,85 +1,101 @@
-# Firestarter
+[subvisual]: http://subvisual.co/
+[build-page]: https://travis-ci.org/subvisual/firestarter
+[code-climate]: https://codeclimate.com/github/subvisual/firestarter
 
-[![Build Status](https://travis-ci.org/subvisual/firestarter.svg?branch=master)](https://travis-ci.org/subvisual/firestarter) [![Code Climate](https://codeclimate.com/github/subvisual/firestarter/badges/gpa.svg)](https://codeclimate.com/github/subvisual/firestarter)
+Firestarter
+===========
 
-Firestarter is the base Rails application used at [Group Buddies](http://groupbuddies.com).
+[![Build Status](https://travis-ci.org/subvisual/firestarter.svg?branch=master)][build-page]
+[![Code Climate](https://codeclimate.com/github/subvisual/firestarter/badges/gpa.svg)][code-climate]
+
+Firestarter is the Rails application generator used at [Subvisual][subvisual].
+
+**Table of Contents**
+
+* [Installation](#installation)
+* [Gemfile](#gemfile)
+* [Other goodies](#other-goodies)
+* [Git](#git)
+* [Dependencies](#dependencies)
+* [Issues](#issues)
+* [Contributing](#contributing)
+* [Credits](#credits)
+* [License](#license)
 
 Installation
 ------------
 
-First install the firestarter gem:
+First install the Firestarter gem:
 
     gem install gb-firestarter
 
 Then run:
 
-    firestarter projectname
+    firestarter <project_name>
+
+This will create a new Rails application in `./<project_name>`. By default,
+Firestarter will use Rails 4.2.0 to generate new applications. You can override
+this by setting the `RAILS_VERSION` environment variable when running the
+generator:
+
+    RAILS_VERSION=4.1.16 firestarter <project_name>
 
-This will create a Rails app in `projectname`.
 
-By default this script creates a new git repository. See below if you
-want to use it against an existing repo.
+By default this script creates a new git repository. See below if you want to
+use it against an existing one.
 
 Gemfile
 -------
 
-To see the latest and greatest gems, look at Firstarter's
-[Gemfile](templates/Gemfile.erb),
-which will be appended to the default generated projectname/Gemfile.
+To see the latest and greatest gems, look at Firestarter's template
+[Gemfile](templates/Gemfile.erb), which will be appended to the default
+generated `<project_name>/Gemfile`.
 
 It includes application gems like:
 
-* [Bourbon](https://github.com/thoughtbot/bourbon) for Sass mixins
-* [Email Validator](https://github.com/balexand/email_validator) for email
-  validation
-* [jQuery Rails](https://github.com/rails/jquery-rails) for jQuery
-* [Postgres](https://github.com/ged/ruby-pg) for access to the Postgres database
```

Comment:
No, you can see it as an added line too. Probably has to do with that `;` that I added and caused a line break. Git considers I removed that line and added two new ones.